examples/kube-stack-private/README.md

```diff
@@ -0,0 +1,103 @@
+## Creating a private Kubernetes Cluster using Terraform
+
+### Purpose of This Example
+The purpose of this example is to show you how to go through the process of building AWS infrastructure with FPCO's AWS Terraform Foundation and then install Kubernetes on that infrastructure using Kubespray.
+
+### How to Use This Example
+This example has two stages. In the first stage we will use FPCO Terraform modules to create a cluster on AWS. Once the cluster is up and running properly, we will use Kubespray to install Kubernetes inside that cluster. Please note, that Kubespray in the Kubernetes Incubator is a work in progress. The state of any given master branch might have a problem that causes the Kubespray installation to fail at some point. Nonetheless, the steps below should at least get to you to the point of an operational Kubernetes cluster. We hope to provide a stable version of Kubespray soon, which you can use with this example.
+
+#### Create the cluster using FPCO Terraform modules
+
+First, review the variables defined in `variables.tf`. You can change their values by adding corresponding values in `vars.env` and `terraform-tfvars.tpl` as per the 3 examples currently in `vars.env`. Be sure to review the current values as well. In particular it is likely necessary that you choose a new value for the CoreOS AMI by choosing an HVM AMI for your particular region from the list for the stable channel found [here](https://coreos.com/os/docs/latest/booting-on-ec2.html) or [here](https://goo.gl/wLhUyH"). The version currently there is likely out of date.
+
+Once you've set up your variables, run these `make` targets:
+
+```bash
+ᐅ make render-tfvars
+ᐅ make network
+ᐅ make plan
+ᐅ make apply
+```
+
+After you have successfully completed creating the cluster infrastructure, add the following commands to your ssh configuration file `~/.ssh/config`:
+```
+Host 10.10.*
+  ProxyCommand ssh -W %h:%p bastion
+  IdentityFile ~/.ssh/id_rsa
+  User core
+Host ip-10-10-*
+  ProxyCommand ssh -W %h:%p bastion
+  IdentityFile ~/.ssh/id_rsa
+  User core
+Host bastion
+  hostname <your bastion ip here>
+  User ubuntu
+  IdentityFile ~/.ssh/id_rsa
+  ControlMaster auto
+```
+Be sure to substitute the IP address for the bastion host. If you aren't sure what that is, run the command `terraform show` and look for `bastion_ip`. If you used a different ssh key than the standard one, be sure to edit the config to reflect the key you actually used.
+
+##### Install Kubernetes Using Kubespray
+
+You are now ready to use Kubespray to install Kubernetes in the cluster you just created. Follow the following steps:
+1. Use `pip` to install the following Python packages: `boto3`,`os`,`json`,`argparse`,`netaddr`
```

Comment:
this should be done in a python virtualenv. I can lookup a `Makefile` that includes this.

Reply:
Since this is akin to a "getting started guide" not guidance for production best practices, do we really want to get into virtualenv? Moreover, these steps are not part of a Makefile. Perhaps they should be, but for that, it seems we should wait till we have a firm standard version of Kubespray.